src/app/pwa/(pages)/profile/_components/onramps/unlimit.tsx

```diff
@@ -2,6 +2,8 @@ import { Button } from '@/app/pwa/_components/ui/button'
 import { GateFiDisplayModeEnum, GateFiSDK } from '@gatefi/js-sdk'
 import { useEffect, useRef, useState } from 'react'
 import { useAccount } from 'wagmi'
+import { inProduction } from '@/app/pwa/_lib/utils/environment.mjs'
+import { inDevelopment } from '@/app/pwa/_lib/utils/environment.mjs'
```

Comment:
We should take care of cleaning unused imports, we can automate it with eslint rules.